hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -171,8 +171,9 @@ public class HoodieWriteConfig extends HoodieConfig {
           + "Refer to org.apache.spark.storage.StorageLevel for different values");
 
   public static final ConfigProperty<String> PRECOMBINE_FIELD_NAME = ConfigProperty
-      .key("hoodie.datasource.write.precombine.field")
+      .key("hoodie.datasource.write.precombine.fields")
```

Comment:
-> hoodie.datasource.write.ordering.fields ?

Reply:
should we just remove this write config option because it is never a write config, we can check the old write config option when creating table the first time. for all the other cases, we should read from the table config to get the ordering fields instead.